.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,25 @@
+{
+	"name": "wasmCloud",
+	"image": "rust:bullseye",
```

Comment:
nit; The toolchain version and target set is already managed in `rust-toolchain.toml`, is there a way we could avoid relying on a particular version and just use something that has `rustup` installed and let it figure stuff out?